src/app/shared/dockstore.model.ts

```diff
@@ -1,15 +1,19 @@
 export class Dockstore {
+  static readonly HOSTNAME = 'http://localhost';
```

Comment:
Maybe add a comment here indicating that a user should fill this in with something more appropriate for login/integration purposes